zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java

```diff
@@ -91,6 +92,11 @@ public static String getPrincipal() {
     String principal;
     if (subject.isAuthenticated()) {
       principal = extractPrincipal(subject);
+      if (ZeppelinServer.notebook.getConf().isUsernameForceLowerCase()) {
+        log.debug("Converting principal name " + principal
+          + " to lower case:" + principal.toLowerCase());
```

Comment:
log.warn()? since this is possibly security related.

Reply:
Can do but then there will be lot noise at info log level. Are we sure that we want to have this?